SystemInformer/thrdlist.c

```diff
@@ -1295,6 +1347,7 @@ BOOLEAN NTAPI PhpThreadTreeNewCallback(
                     SORT_FUNCTION(Created), // Timeline
                     SORT_FUNCTION(ApartmentType),
                     SORT_FUNCTION(ApartmentFlags),
+                    SORT_FUNCTION(HasRpc),
```

Comment:
This is going to disable sorting of all thread columns? The array index for the sort function is the same as the column index and `"SORT_FUNCTION(HasRpc)"` needs to be the last entry since `PH_THREAD_TREELIST_COLUMN_RPC` is the last column.

Reply:
Ahh, yes, my bad. I initially inserted the column right after COM flags but then decided to move it to the end to avoid reordering, and ended up making it worse. Good that you caught it 👍